tests/test_web_websocket.py

```diff
@@ -178,25 +178,29 @@ def test_write_non_prepared():
 def test_can_prepare_ok(make_request):
     req = make_request('GET', '/', protocols=True)
     ws = WebSocketResponse(protocols=('chat',))
+    assert bool(ws.can_prepare(request=req)) is True
```

Comment:
Don't change existing tests but create a new one.

You don't need to call `.can_prepare()` -- just create an instance of `WebSocketReady` and test it.
I expect about three or four new tests